.github/mergify.yml

```diff
@@ -1,6 +1,12 @@
 queue_rules:
+  - name: urgent
+    queue_conditions:
```

Comment:
what's the difference between the `queue_conditions` and `conditions` property?

Reply:
Ah good catch, it looks like [`queue_conditions`](https://docs.mergify.com/configuration/file-format#:~:text=To%20validate%20your%20Mergify%20configuration,tab%20of%20your%20pull%20request.) is the canonical field here. My best guess is that `conditions` is a valid backwards-compatible field that mergify still supports hence why the bot did not freak out about the config being invalid. Updated to use `queue_conditions` now.

Reply:
I double-checked that this config is valid in the editor fwiw :)
<img width="1400" alt="Screenshot 2023-09-28 at 8 11 02 PM" src="https://github.com/ethereum-optimism/optimism/assets/21288394/63002709-75e1-4120-86a2-3ec3ff35a8ce">

Reply:
I think conditions is more general. you can do things with mergify such as make a comment under certain conditions or rebase a pr based on certain condtitions